### PR TITLE
Jmac add Winston

### PR DESCRIPTION
**Pull request by jmacAJ**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -195,14 +201,14 @@ const configFile = config.get('cortexConfigFile');
...
 // Load config file
 if (configFile && fs.existsSync(configFile)) {
-    console.log('Loading config from', configFile);
+    logger.info(`Loading config from ${configFile}`);
     config.loadFile(configFile);
 } else {
     const openaiApiKey = config.get('openaiApiKey');
     if (!openaiApiKey) {
-        throw console.log('No config file or api key specified. Please set the OPENAI_API_KEY to use OAI or use CORTEX_CONFIG_FILE environment variable to point at the Cortex configuration for your project.');
+        throw logger.info('No config file or api key specified. Please set the OPENAI_API_KEY to use OAI or use CORTEX_CONFIG_FILE environment variable to point at the Cortex configuration for your project.');
```
Comment (ademgcl) [inline config.js:209]: logger level warn/error?

File: config.js [hunk 2]
```diff
@@ -263,12 +269,12 @@ const buildModels = (config) => {
 
     // Check that models are specified, Cortex cannot run without a model
     if (Object.keys(config.get('models')).length <= 0) {
-        throw console.log('No models specified! Please set the models in your config file or via CORTEX_MODELS environment variable to point at the models for your project.');
+        throw logger.info('No models specified! Please set the models in your config file or via CORTEX_MODELS environment variable to point at the models for your project.');
```
Comment (ademgcl) [inline config.js:272]: warn/error?

File: server/plugins/azureCognitivePlugin.js [hunk 3]
```diff
@@ -134,11 +135,11 @@ class AzureCognitivePlugin extends ModelPlugin {
             if (API_URL) {
                 //call helper api to mark processing as completed
                 const res = await axios.delete(API_URL, { params: { requestId } });
-                console.log(`Marked request ${requestId} as completed:`, res.data);
+                logger.info(`Marked request ${requestId} as completed: ${res.data}`);
                 return res.data;
             }
         } catch (err) {
-            console.log(`Error marking request ${requestId} as completed:`, err);
+            logger.info(`Error marking request ${requestId} as completed: ${err}`);
```
Comment (ademgcl) [inline server/plugins/azureCognitivePlugin.js:142]: warn/error?

File: server/plugins/azureCognitivePlugin.js [hunk 4]
```diff
@@ -24,7 +25,7 @@ class AzureCognitivePlugin extends ModelPlugin {
             }
             return JSON.parse(await callPathway(this.config, 'embeddings', { text }))[0];
         }catch(err){
-            console.log(`Error in calculating input vector for text: ${text}, error: ${err}`);
+            logger.info(`Error in calculating input vector for text: ${text}, error: ${err}`);
```
Comment (ademgcl) [inline server/plugins/azureCognitivePlugin.js:28]: warn/error?

**Review by ademgcl — APPROVED** [review]

File: config.js (hunk 1 above)
Comment (jmacAJ) [reply config.js:209]: Agreed.

File: config.js (hunk 2 above)
Comment (jmacAJ) [reply config.js:272]: Agreed

File: server/plugins/azureCognitivePlugin.js (hunk 3 above)
Comment (jmacAJ) [reply server/plugins/azureCognitivePlugin.js:142]: Agreed

File: server/plugins/azureCognitivePlugin.js (hunk 4 above)
Comment (jmacAJ) [reply server/plugins/azureCognitivePlugin.js:28]: Agreed

**Pull request merged**